lib/rucio/tests/test_tpc.py

```diff
@@ -62,8 +62,8 @@ def list_fts_transfer(timeout=60, max_attempts=10):
     attempt = 1
 
     time_start = time.time()
-    while running_time < timeout or attempt < max_attempts:
-        rcode, out = run_cmd_process("/usr/bin/python2 /usr/bin/fts-rest-transfer-list -v -s https://fts:8446", timeout=30)
+    while not(running_time >= timeout and attempt > max_attempts):
```

Comment:
`running_time < timeout and attempt <= max_attempts` ? Also `max_attempts` => `min_attempts`

Reply:
you're right, it expands to the same logic.